patches/chrome-renderer-media-chrome_key_systems_provider.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/renderer/media/chrome_key_systems_provider.cc b/chrome/renderer/media/chrome_key_systems_provider.cc
+index e1ba75626fea8e5018964a51ff12561a41cc75e5..13e9c18f0b0fe742d482815e7a4e75bb63245488 100644
+--- a/chrome/renderer/media/chrome_key_systems_provider.cc
++++ b/chrome/renderer/media/chrome_key_systems_provider.cc
+@@ -8,7 +8,7 @@
+ #include "chrome/renderer/media/chrome_key_systems.h"
+ #include "third_party/widevine/cdm/buildflags.h"
+ 
+-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
++#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
```

Comment:
I don't understand why we're patching this?

Reply:
W/o this patch in this file, reloading doesn't enable widevine after user accepts.